CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
-## [0.18.0](https://github.com/redhat-developer/vscode-quarkus/milestone/20?closed=1) (August 26, 2024)
+## [1.18.1](https://github.com/redhat-developer/quarkus-ls/milestone/22?closed=1) (August 29, 2024)
```

Comment:
I think this should point to vscode-quarkus instead of quarkus-ls

Reply:
Thanks, fixed.